app-design-v2

PooyaDeperson opened the pull request:
look for folder `app-design-v2`

aaroncherian committed:
Update pyproject.toml (#669)

* Update pyproject.toml

* update to `libgl1` to get nox testing working

jonmatthis committed:
Bump version v1.5.2 -> v1.5.3

philipqueen committed:
Fix documentation links (#671)

* Fix documentation links

* change libegl-mesa downloads to libgl1

philipqueen committed:
Patch: Fix Butterworth output not making it to final data (#675)

replace hasattr check on dictionary with "in"

philipqueen committed:
Add data regression warning to welcome dialog (#677)

* Add data regression warning to welcome dialog

* move warning to new dialog

* please linter

* fancy up the message formatting, and add skelly-sweat logo

* opted to show the Warning by default rather than making it contingent on the Welcome screen in order to avoid compex logic

---------

Co-authored-by: jonmatthis <jonmatthis@gmail.com>

jonmatthis committed:
Add bumpver as dev dep

jonmatthis committed:
Bump version v1.5.3 -> v1.5.4

aaroncherian committed:
Update README.md (#680)

* Update README.md

* Update README.md

aaroncherian committed:
Sort videos when getting paths (#682)

* delete workflows

* Create diagnostics.yml

* Update diagnostics.yml

* Update diagnostics.yml

* Update diagnostics.yml

* Update diagnostics.yml

* test update

* trying test data download and run

* fix path to diagnostics

* activate virtual env

* adding diagnostic calculator

* Update diagnostics.yml

* fix bug

* fix path bug

* adding report

* update artifact upload version

* forgot .py oops

* trying out windows optimized calibration?

* fixing paths for windows calibration

* trying another fix

* trying windows data for test data

* duct taping a fix for the github url sample data

* oops

* forgot to fix proper recording path link

* change to windows latest

* changing env activation for windows

* still messed up the venv sigh

* trying to fix encoding issue?

* apparently the recording path also needs changing for windows

* trying windows system with figshare test data

* trying windows cal rearranged into ubuntu on ubuntu

* trying to fix version order in ubuntu plot

* another try?

* and again

* trying packaging versioning

* trying another solution

* clean up

* try to combine steps

* trying to get multi system working

* have sample data for separate systems

* adding calibrate step to test_sample_data

* need to create recording model first

* meaningless change to force action run

* Mutli system action? Could it be?

* testing out non-env to break windows

* re-fixing

* attempt #2 multi system

* oops

* fixing macOS path

* Update calculate_diagnostics.py

* adding numpy and calibration toml upload

* fixing calibration toml path

* calibration fix round 2

* manually trigger action

* explicitly passing the cal toml path

* trying to sort video paths

* move sorting earlier

* sorting more things

3d data was off - adding more sorting to see if there's an issue somewhere else with the name order

* no more sorting

* get rid of pin camera

* Update anipose_camera_calibrator.py

* both sorting and commenting out paths

* reverting to no calibration changes

* trying to check path order across versions

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* Update path_check.yml

* add numeric sorting to paths

* Update get_video_paths.py

* Comment out `pin_camera_0_to_origin'

* add sorting

* linting fix

* trying out sorted paths across repos

* Update dev_diagnostics_multi_system.yml

* Update dev_diagnostics_multi_system.yml

* forgot to re-activate pin camera 0 to origin

* Trying out data with a single calibration toml across all systems

* fixing path bug

* turn off pin to 0

* turn pin to 0 on

* updating artifact upload structure

* more updating

* updatre dev to match

* trying to fix errors

* Update butterworth_warning_dialog.py

* fix linting errors v2

* Update pyproject.toml

* Revert "Merge branch 'test_video_sorting' into update_video_sorting"

This reverts commit 5e6961eb60be464274aca79a5bd9e22fbd9e8780, reversing
changes made to 73e68b3cbdea785f1ca352098ca6a0ca7ae6aabc.

* update skelly_synchronize version

* fix sample and test data calibration files

---------

Co-authored-by: philipqueen <queen.philip@gmail.com>

aaroncherian committed:
Update skellycam and skelly_synchronize versions (#666)

* Update skellycam and skelly_synchronize versions

* Fix skellycam duplicate

---------

Co-authored-by: philipqueen <queen.philip@gmail.com>

trentwirth committed:
QUICK FIX - Update CITATION.cff (#688)

Update CITATION.cff

fixed broken zenodo link, fixed family/given name mix up for Trent & Endurance

gonewest818 committed:
detect variations of "Blender.app" on macOS (#687)

Co-authored-by: Neil Okamoto <neilo@digitalfish.com>

aaroncherian committed:
Aaron/fix pin to 0 code (#686)

* Update anipose_camera_calibrator.py

* Update anipose_camera_calibrator.py

* add rotation zeroing out as well

* formatting changes/make one umbrella function

* update for code clarity

* linting fix

aaroncherian committed:
Update skellytracker (#690)

jonmatthis committed:
Bump version v1.5.4 -> v1.6.0

aaroncherian committed:
Patch progress_callback bug in AniposeCameraCalibrator (#696)

* Update anipose_camera_calibrator.py

* Revert "Update anipose_camera_calibrator.py"

This reverts commit 663c00929ccff5417ef15837d6078f0e03811d55.

* set default callable

---------

Co-authored-by: jonmatthis <jonmatthis@gmail.com>

jonmatthis committed:
Bump version v1.6.0 -> v1.6.1

jonmatthis committed:
Jon/add release notes dialog (#694)

* init release notes dialog

* chktpot

* good start for release notes tab thing

* good start to the tab release  notes thing

* good enough

* update

* late commit

* update release notes pop up logic

* linting fixes

* release notes widget

---------

Co-authored-by: MocapMike <mic.nguyen@northeastern.edu>
Co-authored-by: Aaron Cherian <aaron.cherian2@gmail.com>

jonmatthis committed:
Bump version v1.6.1 -> v1.6.2

jonmatthis committed:
Merge branch 'main' of https://github.com/freemocap/freemocap

jonmatthis committed:
tweaks for release notes popup

jonmatthis committed:
tweak for 1.6.3

jonmatthis committed:
Bump version v1.6.2 -> v1.6.3

aaroncherian committed:
Add calibration diagnostics (#700)

* adding calibration diagnostics

* reorganize/add csv

* get rid of csv finding

* change stats to basemodel

* cast to pathlib path

* fix diagnostics linter issues

* fix all the other linting issues

* security changes

aaroncherian committed:
Update sample data hosting (#701)

* Update download_sample_data.py

* rework sample data downloading

* fix linting

* update action to use new sample data downloading

* use enums

philipqueen committed:
Patch: update deprecated pydantic `dict` in data saver models (#711)

replace .dict with  model_dump for pydantic 2

philipqueen committed:
Better ffmpeg missing behavior (#704)

* add warning and don't attempt synchronization if ffmpeg isn't found

* fix warning label

* Better label visibility

---------

Co-authored-by: jonmatthis <jonmatthis@gmail.com>

aaroncherian committed:
Patch: update sample data for tests (#712)

Update conftest.py

aaroncherian committed:
Charuco groundplane (#691)

* initial start

* Update anipose_camera_calibrator.py

* working alignment behind a bool

* file name updates/lint fix

* rename groundplane bool

* create separate function/move 2d tracking to separate function

* clean up backend code

* Update anipose_camera_calibrator.py

* linting fixes

* basic checkbox added

* autocalculate basis vectors based on board x/y square lengths

* Update charuco_groundplane_utils.py

* adding second bool to frontend

* fixing checkbox visibility

* moving rotate around 90 to just be used in single cam

* linting fixes

* adding charuco board choice as an option

* even more linting fixes

* fix default sizing so options overlap less on opening

* fix overwriting of hbox_top in camera controller group box

* auto_frame_finding + graceful failure

* Update anipose_camera_calibrator.py

* linting fixes

* save out 3d board position

* auto set num processes

* adding dialog popup

* fix linting

* fix action for groundplane board changes

* fix for action pt. 2

* Update run_calibration.py

* use skellytracker for charuco detection in anipose (#695)

* use skellytracker for charuco detection in anipose

* Error check on getting charuco data

* cast to float64 for groundplane calculations

* make the boards detected output more clear

* update skellytracker version to bring in `process_list_of_videos`

* Pass board parameters to skellytracker

* remove board size check

* remove completed TODO

---------

Co-authored-by: Aaron Cherian <aaron.cherian2@gmail.com>

* add 'groundplane_calibration' bool

* Update charuco_groundplane_utils.py

* Update charuco_groundplane_utils.py

* interpolate the charuco data

* Update charuco_groundplane_utils.py

* small UI change

* update to new blender

---------

Co-authored-by: MocapMike <mic.nguyen@northeastern.edu>
Co-authored-by: philipqueen <queen.philip@gmail.com>

paulmatthis committed:
Update CONTRIBUTING.md (#725)

Splitting off the org/non-org instructions from 
updating docs" guidelines at https://freemocap.github.io/documentation/updating-documentation.html#if-you-are-a-member-of-the-freemocap-organization

aaroncherian committed:
Update anipose_camera_calibrator.py (#723)

* Update anipose_camera_calibrator.py

* add setters/getters to cam/cam ground

---------

Co-authored-by: jonmatthis <jonmatthis@gmail.com>

emmanuel-ferdman committed:
Update deprecated pydantic dict method (#721)

Signed-off-by: Emmanuel Ferdman <emmanuelferdman@gmail.com>
Co-authored-by: philipqueen <queen.philip@gmail.com>

aaroncherian committed:
fix charuco board saving (#728)

Update anipose_camera_calibrator.py

philipqueen committed:
Expose multiprocessing to batch processing (#726)

PooyaDeperson committed:
Add Tiled PDF for Charuco Board (3×5 and 5×7) (#733)

* Add Tiled PDF for Charuco Board (3×5 and 5×7)

This PR adds tiled PDF files for assembling Charuco boards at sizes 3×5 and 5×7.
.
.
Instructions:

Print odd-numbered pages only.

Skip the even-numbered black pages — instead, use black cardboard or plain black paper.

Mount the printed tiles on a thick cardboard backing, following the reference layout.

Ensure all pages are printed at the same scale and trimmed precisely as indicated.

* update-tiled-charuco-board

This PR adds tiled PDF files for assembling Charuco boards at sizes 3×5 and 5×7
new- updated the 3x5 .pdf

* update-on-instructions

update-on-instructions and removed-unnecessary-webp-images

philipqueen committed:
Cleanup groundplane UI (#735)

* fix charuco labels

* sync calibration options between widgets

* fix auto updating with textChanged -> textEdit

* linting

* shorten camera controller text to fit in UI

* remove redundant field in scss

* only display charuco options when calibrate form active recording option is chosen

* small UI cleanup

* lint, remove todo

---------

Co-authored-by: Aaron Cherian <aaron.cherian2@gmail.com>
Co-authored-by: jonmatthis <jonmatthis@gmail.com>

PooyaDeperson committed:
adding-app-design-v2-initial

PooyaDeperson committed:
Merge pull request #1 from PooyaDeperson/app-design-v2

builiding-app-design-v2-initials

PooyaDeperson committed:
layout-initial html css

PooyaDeperson committed:
update on the layout

PooyaDeperson committed:
refactoring css to util css mode

PooyaDeperson committed:
update on css util mode

PooyaDeperson committed:
update on css utils border styles

PooyaDeperson committed:
color var update

PooyaDeperson committed:
fixing resposive issues and overflowing content

PooyaDeperson committed:
scrollbars styles designs

PooyaDeperson committed:
adding overflow- min and max height

PooyaDeperson committed:
fix on height to width

PooyaDeperson committed:
upadting the dom strcuture

PooyaDeperson committed:
updat ehtml

PooyaDeperson committed:
updating css structure

PooyaDeperson committed:
adjusting layout

PooyaDeperson committed:
html udpates

PooyaDeperson committed:
fix border color

PooyaDeperson committed:
fix=border-600 color

PooyaDeperson committed:
adding footer info and action container styles and other elements

PooyaDeperson committed:
Merge pull request #2 from PooyaDeperson/adding-layout-foundation

layout-initial html css

PooyaDeperson committed:
Update README.md

Signed-off-by: Pooya Deperson <poamrd@gmail.com>

PooyaDeperson committed:
scrollbars design

PooyaDeperson committed:
adding- correct overflow styles

PooyaDeperson committed:
Merge pull request #3 from PooyaDeperson/adding-layout-foundation

scrollbars design

PooyaDeperson committed:
adding-modal---icon-setup-css---hover-transitions---removed-conflict-css-in-indexcss

PooyaDeperson committed:
rename splash screen tsx

PooyaDeperson committed:
Merge pull request #4 from PooyaDeperson/adding-splash-screen-modal

adding-modal---icon-setup-css---hover-transitions---removed-conflict-…

PooyaDeperson committed:
splash-screen-style-and-images

PooyaDeperson committed:
splash-screen-styles-and icons

PooyaDeperson committed:
rename icons

PooyaDeperson committed:
splash-screen-button and states and update on css

PooyaDeperson committed:
fixing-img-src- link

PooyaDeperson committed:
Merge pull request #5 from PooyaDeperson/adding-splash-screen-modal

Adding splash screen modal

PooyaDeperson committed:
splash color and font adjustments

PooyaDeperson committed:
Merge pull request #6 from PooyaDeperson/adding-splash-screen-modal

splash color and font adjustments

PooyaDeperson committed:
adding-checkbox- and hyperlink styles

PooyaDeperson committed:
checkbox styles

PooyaDeperson committed:
splash-screen-styles-and css optmizations

PooyaDeperson committed:
Merge pull request #7 from PooyaDeperson/adding-splash-screen-modal

Adding splash screen modal styles and css optimization + adding other components in ui components ui tsx

PooyaDeperson committed:
adding-meaningful-comments for other devs

PooyaDeperson committed:
Merge pull request #8 from PooyaDeperson/adding-splash-screen-modal

adding-meaningful-comments for other devs

PooyaDeperson committed:
adding-css active scale for buttons

PooyaDeperson committed:
ignoring checkbox from the acttive interaction

PooyaDeperson committed:
adding-additional css scale on active and css class clarity

PooyaDeperson committed:
Merge pull request #9 from PooyaDeperson/adding-splash-screen-modal

Adding splash screen modal

PooyaDeperson committed:
changing css for typography

PooyaDeperson committed:
Merge pull request #10 from PooyaDeperson/adding-splash-screen-modal

changing css for typography

PooyaDeperson committed:
adding comma

PooyaDeperson committed:
adding : press esc key to hide splash screen popup , making card buttons as components to reuse

PooyaDeperson committed:
Merge pull request #11 from PooyaDeperson/adding-splash-screen-modal

Adding splash screen modal

jonmatthis committed:
Jon/last polish before 1.7.0 (#741)

* everything but the release notes popup

* add plain 5x3 board

* updated release notes (will replace ai text)

* tmp 170 notes

* should be gtg?

* pycharm autoformat

* make linter happy

* fix charuco size label

jonmatthis committed:
Bump version v1.6.3 -> v1.7.0

PooyaDeperson committed:
repositioning-checkbox props in the file

PooyaDeperson committed:
added-segmented-control for modes

PooyaDeperson committed:
fix-segmented-passing-functions-on-click

PooyaDeperson committed:
adding-2mode-info modes and live and post process modes using segmented control

PooyaDeperson committed:
Merge pull request #12 from PooyaDeperson/adding-segmented-controls-capture-and-post-modes

Adding segmented controls capture and post modes

PooyaDeperson committed:
adding-header-toolbar-state button and right section support and help

PooyaDeperson committed:
formatting code

PooyaDeperson committed:
changing icon sizes

PooyaDeperson committed:
adding-toggles styles

PooyaDeperson committed:
Merge pull request #13 from PooyaDeperson/adding-toggle-component

Adding toggle component

PooyaDeperson committed:
styles for ubaction contatiner, scroll behaviour

PooyaDeperson committed:
adding-fullwidth- button and other icons

PooyaDeperson committed:
adding- extra classes -

PooyaDeperson committed:
Merge pull request #14 from PooyaDeperson/adding-toggle-component

Adding toggle component, extra styles and more properties elements

PooyaDeperson committed:
Merge branch 'main' into main

PooyaDeperson committed:
adding-dropddown-component-and-interations-+ download icon

PooyaDeperson committed:
Merge pull request #15 from PooyaDeperson/adding-dropdpwn-interactions

adding-dropddown-component-and-interations-+ download icon

PooyaDeperson committed:
adding relevant seo and fixing font size on toggle

PooyaDeperson committed:
Merge pull request #16 from PooyaDeperson/adding-dropdpwn-interactions

adding relevant seo and fixing font size on toggle

PooyaDeperson committed:
adding text overflow hidden for toggle and adding flex-1 basis styles for main containters

PooyaDeperson committed:
adding animation styles and variables in css

PooyaDeperson committed:
applying slide-down animation on the dropdown container

PooyaDeperson committed:
adjusting the padding of the splash screen action container

PooyaDeperson committed:
adding-toggle-hide-and show logics

PooyaDeperson committed:
adding-skip calibration toggle to show and hide the record action group

PooyaDeperson committed:
adding opacity and disable interactions instead of hiding the elements for the conditional toggles

PooyaDeperson committed:
adding extra ux logic for the toggle to turn off and on the child when toggle off

PooyaDeperson committed:
fix font import syntax error, adding text bg-md style for caption

PooyaDeperson committed:
cleaning css

PooyaDeperson committed:
Merge pull request #17 from PooyaDeperson/adding-interaction-logics

Adding interaction logics

PooyaDeperson committed:
update font size css

PooyaDeperson committed:
changing css for disabled state

PooyaDeperson committed:
more css updates on the disabled state

PooyaDeperson committed:
formatting

PooyaDeperson committed:
camera-tiles styles- active mode header toolbar : stream button and

PooyaDeperson committed:
adding toggle button component and streaming camera interaction

PooyaDeperson committed:
adding streaming and websocket connection interactions

PooyaDeperson committed:
Merge pull request #18 from PooyaDeperson/adding-streaming-camera-permissions-logics

Adding streaming camera permissions logics

PooyaDeperson committed:
adding persistent connection dropdpwn logic

PooyaDeperson committed:
adding coonection dropdpwn logic

PooyaDeperson committed:
fixing classnames for connection dropdpwn

PooyaDeperson committed:
Merge pull request #19 from PooyaDeperson/persistent-connection-logic

Persistent connection dropdpwn logic

PooyaDeperson committed:
adding standalone togglebutton component

PooyaDeperson committed:
Merge pull request #20 from PooyaDeperson/persistent-connection-logic

adding standalone togglebutton component

jonmatthis commented:
TBH, I'm not sure how to handle the changes in the current PR! I wound up copying your `app-design-v2` code into a separate branch and then merging those changes into the `freemocap/development` branch that is the target of this PR. 

Its a bit of a wonky workflow, but I'm sure we can figure out how to work forward with it. To be clear, all of my changes are in the `app-design-v2/` folder in the `freemocap/development` branch here: https://github.com/freemocap/freemocap/tree/development/app-design-v2

PooyaDeperson commented:
Great work! 🤔 
Now, should I copy your changes to my main FreeMoCap repo so we can work asynchronously? 🔄 

We need a workflow where I work on the UI while you make changes on other parts, all within the same repo. 💻 

Should I start copying your changes over now? 🤔 #Workflow #Collaboration #Repo #UI #Changes

jonmatthis commented:
> should I copy your changes to my main FreeMoCap repo so we can work asynchronously? 🔄

I think that works - my updates are a bit clunky because I wound up doing  a bunch of copy-pasting between your fork and things I needed from `skellycam-ui/`

I also realized that I'd mistakenly told you to base your fork/PR on the `freemocap/main` branch, when I SHOULD have said to do so from the `freemocap/development` branch! The `main` branch is so old and out of date at this point that we should pretty much only use it as reference. I swapped this PR over, but now there's a million conflicts that would be annoying to clear. 

I think we can continue to hack around and shoehorn whatever we need to to make it work - we're currently working entirely in the `app-design-v2` folder (which doesn't touch anything else in the `freemocap/` repo), so it's not a huge problem either way. 

> We need a workflow where I work on the UI while you make changes on other parts, all within the same repo. 💻

Yeah, I think the move will be for me to work mostly on the connection between the `/store` and `/services` UI code and the backend server API. I think we should be able to work something out where you can build the UX/UI stuff on your own, and then we can chat about the kind of info you will need to pull from the RTK stores, so I can work on providing that info from the server/websocket. 

Right now, I'm most interested in recreating the core `skellycam` functionality (i.e. connect and configure cameras, stream images through a websocket, and record videos to disk)

PooyaDeperson commented:
Great, thanks. Quick update on my side:

- Made a backup

- Copied over the development branch

- Tested, and I can run it locally now

I can update this PR so we’re synced at this point. 
We can keep working in syncwhat do you think?

PooyaDeperson committed:
exited main branch and updated and synced to refactored ui redux.

PooyaDeperson committed:
merged changes

PooyaDeperson committed:
Merge pull request #21 from PooyaDeperson/update-moved-refactored-to-development

Update moved refactored to development

PooyaDeperson commented:
> > I can update this PR so we’re synced at this point. We can keep working in sync - what do you think?
> 
> I think that works! If you have trouble getting things lined up in this PR, we can close it and make a new one

here done, please check

PooyaDeperson committed:
fix icon class naming

PooyaDeperson committed:
bringing back error icon

PooyaDeperson committed:
bringing back the original codebase

PooyaDeperson committed:
refactored ButtonSM

PooyaDeperson committed:
refactored Checkbox

PooyaDeperson committed:
refactored buttonCard + SplashModal

PooyaDeperson committed:
dropdpwn failed commit

PooyaDeperson committed:
Revert "refactored buttonCard + SplashModal"

This reverts commit 46c19ac697b464a9a331fe6d11ce7387660dbd01.

PooyaDeperson committed:
failed changes

PooyaDeperson committed:
failed changes

PooyaDeperson committed:
reverting

PooyaDeperson committed:
reverting back to original codebase

PooyaDeperson committed:
reverting back to original codebase

PooyaDeperson committed:
adding- infopanel modes segmented control

PooyaDeperson committed:
refactor header to its own file as HeaderPanel

PooyaDeperson committed:
Revert "refactor header to its own file as HeaderPanel"

This reverts commit 50cafa2ba0ecab75c07bae8dba426dc4dd32b913.

PooyaDeperson committed:
removing standalone connection component

PooyaDeperson committed:
refactored-header-section-to-HeaderPanel.tsx

PooyaDeperson committed:
introduced-ModePanel

PooyaDeperson committed:
refactored-modes-into-capture-live-and-post-process

PooyaDeperson committed:
refactore-modes-and renamed new files

PooyaDeperson committed:
Merge pull request #22 from PooyaDeperson/reanming-files-and-reorder

Renaming files and reorder, refactored codebase, introduced mode switch, and introduced panels as separated files also introduce info bottom panel switch

PooyaDeperson committed:
added post-process-3d visualzier layout

PooyaDeperson committed:
adding interactive 3d visualizer

PooyaDeperson committed:
refactored theeDscene

PooyaDeperson committed:
styling threeDscene

PooyaDeperson committed:
Merge pull request #23 from PooyaDeperson/post-process-layout

Post process layout : adding react three fiber : adding threeDscene in the layout with limited interactions

PooyaDeperson committed:
adding 3d asset skelly character

PooyaDeperson committed:
adjust the cmaera position, 3d assets and 3d viz container

PooyaDeperson committed:
Merge pull request #24 from PooyaDeperson/post-process-layout

Post process layout

PooyaDeperson committed:
changing the camera default pos and rot

PooyaDeperson committed:
adding skeleton loader animation tot he 3d visuazlier

PooyaDeperson committed:
Update ThreeDScene.tsx

PooyaDeperson committed:
adding skeleton loader

PooyaDeperson committed:
adding sudlte fadeIn animation to the container modes

PooyaDeperson committed:
Merge pull request #25 from PooyaDeperson/post-process-layout

Post process layout

PooyaDeperson committed:
adding video icon- and re positioning 3d visualizer

PooyaDeperson committed:
adding the correct post process components

PooyaDeperson committed:
adding input interaction tooltip

PooyaDeperson committed:
Merge pull request #26 from PooyaDeperson/post-process-layout

Post process layout

PooyaDeperson committed:
fix on the input selector style and postion

PooyaDeperson committed:
Merge pull request #27 from PooyaDeperson/post-process-layout

fix on the input selector style and postion

PooyaDeperson committed:
changing icons

PooyaDeperson committed:
input states hover and focus interaction

PooyaDeperson committed:
addin subaction groups

PooyaDeperson committed:
styles for subaction group

PooyaDeperson committed:
repositoning the tooltip value selector

PooyaDeperson committed:
dropdpwn selector and css styles

PooyaDeperson committed:
adding logic for yolo crop activation and deactivation

PooyaDeperson committed:
finalizing post processing properties panel

PooyaDeperson committed:
Merge pull request #28 from PooyaDeperson/adding-properties-and-options-components

Adding properties and options components / Properties panel / overflow styles / properties panel toggle and options and logics on enabling and disabling certain options

PooyaDeperson committed:
adding more properties toggles in the post process

PooyaDeperson committed:
Merge pull request #29 from PooyaDeperson/adding-properties-and-options-components

adding more properties toggles in the post process

PooyaDeperson committed:
fxing style issues on the pos process properties

PooyaDeperson committed:
Merge pull request #30 from PooyaDeperson/adding-properties-and-options-components

fxing style issues on the pos process properties

PooyaDeperson committed:
capturelive cameras scroll-y

PooyaDeperson committed:
static file directory settings modal and triggers

PooyaDeperson committed:
file directory modal interaction and correct position

PooyaDeperson committed:
adding subfolder and removing subfolder interaction

PooyaDeperson committed:
Merge pull request #31 from PooyaDeperson/modes-capturelive-post-process-additional-designs

Modes capturelive post process additional designs

PooyaDeperson committed:
adding file directory sync

PooyaDeperson committed:
Merge pull request #32 from PooyaDeperson/modes-capturelive-post-process-additional-designs

adding file directory sync

PooyaDeperson committed:
adding web version folder selection

PooyaDeperson committed:
adding-timestamps format into recording file

PooyaDeperson committed:
adding undercore

PooyaDeperson committed:
Merge pull request #33 from PooyaDeperson/re_adding_timestamps_into_filename

Re adding timestamps into filename

PooyaDeperson committed:
rename file recording

PooyaDeperson committed:
rename subfolder name

PooyaDeperson committed:
Merge pull request #34 from PooyaDeperson/re_adding_timestamps_into_filename

Re adding timestamps into filename

PooyaDeperson committed:
sync subfolder name to capture live main window : fix long name issues on the settings modal

PooyaDeperson committed:
Merge pull request #35 from PooyaDeperson/re_adding_timestamps_into_filename

sync subfolder name to capture live main window : fix long name issue…

PooyaDeperson committed:
streaming all cameras : known issues: stream camera is not activated

PooyaDeperson committed:
streaming cameras

PooyaDeperson committed:
Merge pull request #36 from PooyaDeperson/camera-permissions-settings-and-other-camera-styles

Camera permissions settings and other camera styles

PooyaDeperson committed:
stream button logic to connected state

PooyaDeperson committed:
Merge pull request #37 from PooyaDeperson/camera-permissions-settings-and-other-camera-styles

stream button logic to connected state

PooyaDeperson committed:
camera settings and video feed adjustments

PooyaDeperson committed:
camera video containers ui styles and camera- settings modal

PooyaDeperson committed:
restyling and repositioning the camera settings modal

PooyaDeperson committed:
adding camera name checkbox label to check and uncheck

PooyaDeperson committed:
styles for checkbox and camera feed

PooyaDeperson committed:
Merge pull request #38 from PooyaDeperson/camera-permissions-settings-and-other-camera-styles

Camera permissions settings and other camera styles

PooyaDeperson committed:
added draggable ui elements and

PooyaDeperson committed:
warning tooltip and checkbox sttyles mismatch

PooyaDeperson committed:
camera action container adjustment

PooyaDeperson committed:
camera feed adjustments / excluded camera tooltip /

PooyaDeperson committed:
checkbox styles

PooyaDeperson committed:
Merge pull request #39 from PooyaDeperson/camera-permissions-settings-and-other-camera-styles

Camera permissions settings and other camera styles

PooyaDeperson committed:
adds-visual-clearity-for-the-selected-camera-feed-settings

PooyaDeperson committed:
grid layout changes / refactored and added iconsegmented coontrol / new icons / scrollbar color changes

grid layout changes / refactored and added iconsegmented coontrol / new icons / scrollbar color changes

PooyaDeperson committed:
Merge pull request #40 from PooyaDeperson/css-styles-and-interactions-clearity

Css styles and interactions clearity

PooyaDeperson committed:
margin-top added for on video container for outline visibility

PooyaDeperson committed:
fix error typoe

PooyaDeperson committed:
Merge pull request #41 from PooyaDeperson/css-styles-and-interactions-clearity

margin-top added for on video container for outline visibility

PooyaDeperson committed:
uicomponent-general-refactor

PooyaDeperson committed:
removing-uicomponents.tsx

PooyaDeperson committed:
Update readme and file and adding project file structure

PooyaDeperson committed:
updating readme file

PooyaDeperson committed:
Merge pull request #42 from PooyaDeperson/major-refactor

Major refactor and updating the readme file

PooyaDeperson committed:
Update README.md

Signed-off-by: Pooya Deperson <poamrd@gmail.com>

PooyaDeperson committed:
Update README to reflect file changes with icons

Signed-off-by: Pooya Deperson <poamrd@gmail.com>

PooyaDeperson committed:
update readme file

PooyaDeperson committed:
refactor css styles

PooyaDeperson committed:
updateing readme file

PooyaDeperson committed:
Merge branch 'main' into major-refactor

Signed-off-by: Pooya Deperson <poamrd@gmail.com>

PooyaDeperson committed:
fix typo on App.css

PooyaDeperson committed:
Merge branch 'major-refactor' of https://github.com/PooyaDeperson/Freemocap-Product-Design-v2 into major-refactor

PooyaDeperson committed:
Merge pull request #43 from PooyaDeperson/major-refactor

Major refactor styles and other files

PooyaDeperson committed:
adding comments and cc

PooyaDeperson committed:
Merge pull request #44 from PooyaDeperson/adding-comments-usage-and-cc

adding comments and cc

PooyaDeperson committed:
cc changes

PooyaDeperson committed:
Merge pull request #61 from PooyaDeperson/adding-comments-usage-and-cc

cc changes

PooyaDeperson committed:
fix subcat icon path

PooyaDeperson committed:
Merge pull request #62 from PooyaDeperson/adding-comments-usage-and-cc

fix subcat icon path
